index.js

```diff
@@ -37,14 +37,15 @@ const gitClientFactory   = require('./src/git.js');
 const github             = require('./src/github.js');
 const taoInstanceFactory = require('./src/taoInstance.js');
 
-const data          = {};
+const data          = {
+    wwwUser : 'www-data'
```

Comment:
Should we prompt a user to provide `wwwUser ` like it done for `token` and `taoRoot `?
Also there is a comment about adding possibility to provide value for `wwwUser` using CLI arguments. I added this possibility in my branch https://github.com/oat-sa/tao-extension-release/blob/feature/TT-358-extract-release-notes-during-the-release-of-an-extension/index.js#L9

Reply:
Of course we should and we should even make the whole way to run the php command configurable (for example sudo isn't always available or some ppl run the release outside a container when the php command should run it so we can imagine to configure the docker command to run it).
However this PR is to provide a quick way to change it without editing the source code from the current state, and then yes, absolutely we can use a more elaborated way to configure it.
What do you think ?

Reply:
I see. I am agree that force users to update source code is not applicable